website/blog/2020-02-06-react-navigation-5.0.md

```diff
@@ -0,0 +1,145 @@
+
+![Redux Devtools in action](assets/announcing-5.0/redux-devtools.gif)
+
+You don't need to use Redux in your apps for this to work and it works without any extra setup!
+
+### Native Stack Navigator
+
+Traditionally, we have written our navigators in JavaScript for greater customizability. It fits a lot of use cases, but sometimes you want the exact native feel and the performance of native navigation. Now, we have added a new native stack navigator that uses native navigation primitives (`UINavigationController` on iOS and `Fragment` on Android) for navigation using the [`react-native-screens`](https://github.com/kmagiera/react-native-screens) library.
+
+<img src="/blog/assets/android-native-stack.gif" height="530" />
+<img src="/blog/assets/ios-native-stack.gif" height="530" />
+
+### Native backends for Material top tab navigator
+
+Similar to native stack, we also have [new backends](https://reactnavigation.org/docs/material-top-tab-navigator.html#pager) for Material top tab navigator based on [`react-native-viewpager`](https://github.com/react-native-community/react-native-viewpager) and [`ScrollView`](https://facebook.github.io/react-native/docs/scrollview).
+
+```js
+import ViewPagerAdapter from 'react-native-tab-view-viewpager-adapter';
+
+// ...
+
+<Tab.Navigator pager={props => <ViewPagerAdapter {...props} />}>
+  {...}
+</Tab.Navigator>
+```
+
+### Other improvements
+
+In addition to these larger improvements, there are several smaller improvements to fit more use cases and make it easier to do certain tasks:
+
+- Revamped drawer navigator to make customizing the drawer sidebar content easier and more flexible.
+- Simpler API for [`reset` action](https://reactnavigation.org/docs/navigation-prop.html#reset) where you can pass the new state directly instead of a chain of actions.
+- More reliable [`focus` and `blur` events](https://reactnavigation.org/docs/navigation-lifecycle.html) to know when a screen's focus state changes.
+- Integration with [`InteractionManager`](https://facebook.github.io/react-native/docs/interactionmanager) to delay tasks until animation is complete.
+- Better safe area handling with [`react-native-safe-area-context`](https://github.com/th3rdwave/react-native-safe-area-context).
+
+## Upgrading
+
+This is a big release and, while the basic concepts such as nesting are the same, the new API is mostly incompatible with the previous API. We know it can be challenge to upgrade your code base. So we're going to keep supporting React Navigation 4 with bug fixes. We'll accept contributions and keep it compatible with the latest React Native version. The old code will now leave at [`react-navigation/react-navigation-4`](https://github.com/react-navigation/react-navigation-4) on GitHub.
```

Comment:
```suggestion
This is a big release and, while the basic concepts such as nesting are the same, the new API is mostly incompatible with the previous API. We know it can be a challenge to upgrade your code base. So we're going to keep supporting React Navigation 4 with bug fixes. We'll accept contributions and keep it compatible with the latest React Native version. The old code will now leave at [`react-navigation/react-navigation-4`](https://github.com/react-navigation/react-navigation-4) on GitHub.
```